stacks/apps/id.yml

```diff
@@ -366,6 +396,48 @@ Resources:
       SecretAccessKey: !GetAtt SesSendingAccessKey.SecretAccessKey
       Region: !Ref AWS::Region
 
+  IdUploadsBucket:
+    Type: AWS::S3::Bucket
+    # TODO: necessary?
+    # DeletionPolicy: Retain
+    # UpdateReplacePolicy: Retain
```

Comment:
CloudFormation will try to delete a bucket if it's not retained if the resource disappears from the template. S3 will throw an error if you try to delete a bucket that still has any objects in it. AFAIK there's no way to get CFN to empty a bucket before deleting it (without custom resources, and even then that's pretty scary). So retention is added to simply avoid the issue, and leave bucket cleanup to the user, since it's hard to guarantee that a bucket will be empty when CFN wants to delete it.